third_party/amd/include/TritonAMDGPUToLLVM/Passes.td

```diff
@@ -23,14 +23,12 @@ def ConvertTritonAMDGPUToLLVM : Pass<"convert-triton-amdgpu-to-llvm", "mlir::Mod
                              "mlir::tensor::TensorDialect",
                              "mlir::triton::TritonDialect",
                              "mlir::triton::gpu::TritonGPUDialect",
-                             "mlir::triton::nvidia_gpu::TritonNvidiaGPUDialect",
-                             "mlir::ROCDL::ROCDLDialect",
-                             "mlir::NVVM::NVVMDialect"];
+                             "mlir::ROCDL::ROCDLDialect"];
 
     let options = [
-        Option<"computeCapability", "compute-capability",
-               "int32_t", /*default*/"80",
-               "device compute capability">,
+        Option<"arch", "arch",
+               "std::string", /*default*/"\"gfx940\"",
```

Comment:
For the default value, gfx940 is for MI300A. Maybe we should use gfx942 here, which is for MI300X

Reply:
This is only used for the cases where we inoke the pass directly, like via command line. If we construct the pass using C++ it will be overwritten by the arguments passed in when invoking the C++ API. 

Thinking a bit, let's just drop the default and update all tests to be explicit which target we are testing--it's always better to be explicit there to be clear.